### ISSUE TITLE
Debugging "Mismatch between download files and md5sums"

### ISSUE THREAD

**Issue by mMontu** [jaos/slapt-src#1]
I've been using slapt-src for some time now and it is great!

Now I'm facing a problem when trying to install a given app, where slapt-src yields the following error:

`Mismatch between download files and md5sums`

I've used the `-e` option in order to force a new download several times, but the error persists.

Is there some option to increase verbosity, so I could know which file is causing the issue, or some guide on how to debug this kind of problem?

**Comment by jaos** [comment]
That means that the application entry in the SLACKBUILDS.TXT file does not have the same # of files to download as checksums to verify.  Which slackbuild source is this?

**Comment by mMontu** [comment]
The problem happens with [docker slackbuild](https://www.slackbuilds.org/repository/14.2/system/docker/). I couldn't find the SLACKBUILDS.TXT nor where slapt-src stored the app file (which I suppose should have the md5sum compared).

But maybe this is beyond the goal of slapt-src: if things doesn't work nicely, probably it is easier/simpler to run the the SlackBuild manually. When I did that I noticed something that may be related to the problem: the link to the source file points to `v1.8.2.tar.gz`, but it brings the file named `docker-1.8.2.tar.gz`.
